File: Gotham_City/Gotham_City.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int cr = Convert.ToInt32(Console.ReadLine());
if(cr < 5)
Console.WriteLine("I got this!");
else if (cr >4 && cr < 11)
Console.WriteLine("Help me Batman");
else
Console.WriteLine("Good Luck out there!");

        }
    }
}

File: Gotham_City/Gotham_City.java
import java.util.Scanner;

public class Program
{
    public static void main(String[] args) {
    Scanner in = new Scanner(System.in);
    int cr = in.nextInt();
if(cr < 5)
System.out.println("I got this!");
else if (cr >4 && cr < 11)
System.out.println("Help me Batman");
else
System.out.println("Good Luck out there!");
    }
}
